pdr_backend/util/web3_accounts.py

```diff
@@ -0,0 +1,81 @@
+import sys
+from os import getenv
+from typing import List
+from enforce_typing import enforce_types
+
+from eth_account import Account
+from pdr_backend.ppss.web3_pp import Web3PP
+from pdr_backend.contract.token import NativeToken, Token
+from pdr_backend.util.contract import get_address
+from pdr_backend.util.mathutil import to_wei, from_wei
+
+
+@enforce_types
+def create_accounts(n_accounts: int):
+    """
+    @description
+        Create new accounts using the web3 provider and print the private_key and the public_key
+    """
+    # loop through n_wallets
+    for i in range(n_accounts):
+        # pylint: disable=no-value-for-parameter
+        new_account = Account.create()
+        print(f"\nWallet #{i}")
+        print(f"Private Key: {new_account._private_key.hex()}")
+        print(f"Public Key: {new_account.address}")
+
+
+@enforce_types
+def view_accounts(addresses: List[str], web3_pp: Web3PP):
+    """
+    @description
+        View account balances for multiple addresses
+    """
+    # get assets
+    native_token = NativeToken(web3_pp)
+    OCEAN_addr = get_address(web3_pp, "Ocean")
+    OCEAN_token = Token(web3_pp, OCEAN_addr, "OCEAN")
+
+    # loop through all addresses and print balances
+    for address in addresses:
+        native_token_balance = native_token.balanceOf(address)
+        OCEAN_balance = OCEAN_token.balanceOf(address)
+
+        print(f"\nAccount {address}")
+        print(f"Native token balance: {from_wei(native_token_balance)}")
```

Comment:
I think for both of these you can print "wei" after the amount

Reply:
Please note that this is not printing in wei (1e0), but in ether (1e18).

**On that note, I would propose we identify a base unit of measurement to report everything in a standard way.** For reports, I'd really like to see everything reported in ether so we can see a meaningful decimal. All other commands right now are reporting the numbers in ether (1e18)

What you're suggesting
![image](https://github.com/oceanprotocol/pdr-backend/assets/69865342/1ee3db62-6caa-4d4f-abee-0374f033fa50)

How it looks right now
![image](https://github.com/oceanprotocol/pdr-backend/assets/69865342/94d91233-ce8a-40de-8f49-b9ba09e97ea2)

Personally, I find the wei numbers useless because there are no meaningful decimals, and no one can count that many units. On the other side I don't recommend we say: "OCEAN Balance: 103.45 eth".

Again, I'd prefer that we simply state somewhere in the cli/apps that all numbers are reported in 1e18, and follow suit.

Cheers!